base/tools/src/main/java/com/netscape/cmstools/tps/config/ConfigModifyCLI.java

```diff
@@ -90,7 +90,7 @@ public void execute(CommandLine cmd) throws Exception {
                 out.println(line);
             }
 
-            configData = ConfigData.fromXML(sw.toString());
+            configData = ConfigData.fromJSON(sw.toString());
```

Comment:
This is going to change the input format for `pki tps-config-modify --input <file>`:
https://github.com/dogtagpki/pki/wiki/PKI-TPS-Configuration-CLI

@06shalini Is it OK if we change this CLI to take a JSON file instead of XML file?

Reply:
@ckelleyRH We'll need to move that page into the source repo so we can maintain different docs for different PKI versions. So PKI 10.11 will continue to use XML, but PKI 11.0 will use JSON.

Reply:
I'm moving the doc into the master branch: https://github.com/dogtagpki/pki/pull/3616. It will need to be cherry-picked into `v10.11` branch as well. Once it's merged, we can update this PR (#3610) to update the doc as well.